packages/collection/__tests__/collection.test.ts

```diff
@@ -770,6 +770,33 @@ describe('sort() tests', () => {
 			const coll = createCollectionFrom(['a', 5], ['b', 3], ['c', 1]);
 			expect(coll.sort()).toStrictEqual(createCollectionFrom(['c', 1], ['b', 3], ['a', 5]));
 		});
+
+		describe('returns correct sort order for test values', () => {
+			const defaultSort = Reflect.get(Collection, 'defaultSort') as (typeof Collection)['defaultSort'];
```

Comment:
You should be able to do this
```suggestion
			const defaultSort = Collection['defaultSort'];
```

Reply:
Was concerned that using `[]` to bypassing access checks for private properties was possibly unintentional and could be removed in the future. However, it looks like it was confirmed to be intentional by the TS team previously, so should be OK. 👍